nntrainer/layers/mol_attention_layer.cpp

```diff
@@ -60,14 +59,14 @@ void MoLAttentionLayer::finalize(InitLayerContext &context) {
     << "MoL Attention layer needs 3-4 inputs.";
 
   auto const &all_dims = context.getInputDimensions();
-  auto const &query_dim = all_dims[AttentionParams::query];
-  auto const &value_dim = all_dims[AttentionParams::value];
-  auto const &state_dim = all_dims[AttentionParams::state];
+  auto const &query_dim = all_dims[MoLAttentionParams::query];
+  auto const &value_dim = all_dims[MoLAttentionParams::value];
+  auto const &state_dim = all_dims[MoLAttentionParams::state];
```

Comment:
So.. we have attention Layer, MoL attention Layer, and Multi-head attention Layer. So.. I guess we could share the attention parameters.. such as AttentionParams::query, AttentionParams::value, AttentionParams::state. and others too.

Reply:
As you can see in https://github.com/nnstreamer/nntrainer/blob/main/nntrainer/layers/mol_attention_layer.cpp#L36 MoL attention needs so many tensors which is not needed in attention and multi head attention layer like u_neg_div, u_pos_div and so on.
So I think it is better to differentiate this.